RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -36,7 +37,7 @@
                                  particleFlowEGammaFinal*
                                  pfParticleSelectionSequence )
 
-particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*particleBasedIsolationSequence)
+particleFlowLinks = cms.Sequence( particleFlow*particleFlowPtrs*chargedHadronIsolation*particleBasedIsolationSequence)
```

Comment:
Strictly speaking the chargedHadronIsolation is not particle-based, but track-based. So I'd rather not change the Egamma sequence, if you agree?